.github/workflows/sync-translations.yml

```diff
@@ -0,0 +1,141 @@
+name: Migrate translations from the old Transifex project
+
+on:
+#  schedule:  # TODO: Maybe before merge
+#    # https://crontab.guru/#0_0_*_*_1
+#    # At midnight on Mondays
+#    - cron: 0 0 * * 1"
+  workflow_dispatch:
```

Comment:
I think we'll keep it as `workflow_dispatch` which means it runs manually in addition to `cron`.